aws/aws_secrets_manager/aws_scm.go

```diff
@@ -68,21 +69,30 @@ func New(
 	if err != nil {
 		return nil, fmt.Errorf("failed to get credentials: %v", err)
 	}
-	config := &aws.Config{
-		Credentials: creds,
-		Region:      &region,
+	credProv := CredentialsToProvider(creds)
```

Comment:
same here

Reply:
Addressed in this [commit](https://github.com/libopenstorage/secrets/pull/76/commits/6a1cf46cb0faa8455010bce92100ac7ce4d99d69)